project1.css

```diff
@@ -39,6 +39,115 @@ section {
   top: 552px;
 }
 
+.add-contact {
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+  width: 375px;
+  height: 800px;
+  left: calc(50% - 375px / 2);
+  top: 1452px;
+  text-align: center;
+}
+
+.add-contact > h1 {
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #fff;
+  padding-top: 70px;
+  width: 375px;
+  position: absolute;
+}
+
+.add-contact > p {
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  color: #ebebff;
+  padding: 0 20px;
+  position: absolute;
+  top: 150px;
+}
+
+.add-contact ul {
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+}
+
+.add-contact li {
+  list-style: none;
+  width: 375px;
+}
+
+input {
+  width: 80%;
+  height: 48px;
+  margin: 0 auto;
+  padding: 5px;
+  border-radius: 8px;
+  margin-bottom: 5px;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+  border: 1px solid #cfd8dc;
+  outline: #cfd8dc;
+}
+
+textarea {
+  height: 170px;
+  border-radius: 8px;
+  width: 80%;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+  border: 1px solid #cfd8dc;
+  outline: #cfd8dc;
+}
+
+form {
+  position: absolute;
+  left: -30px;
+  top: 320px;
+}
+
+form > button {
+  width: 140px;
+  border-radius: 8px;
+  padding: 12px 16px;
+  font-size: 17px;
+  font-weight: 500;
+  outline: none;
+  border: none;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  position: absolute;
+  left: 75px;
+  margin-top: 20px;
+  background: white;
+}
+
+form > button:hover {
+  color: rgb(114, 114, 245);
+}
```

Comment:
- [ ] [OPTIONAL] This was missed by the previous reviewers but still important. Kindly make sure your button has all the interaction as seen on the Figma template. 🙏 

![image](https://user-images.githubusercontent.com/74814780/177572066-84bd8d68-0a9f-451c-b34a-b10417d2f280.png)